pkg/cidata/cidata.TEMPLATE.d/lima.env

```diff
@@ -1,6 +1,7 @@
 LIMA_CIDATA_USER={{ .User }}
 LIMA_CIDATA_UID={{ .UID }}
 LIMA_CIDATA_MOUNTS={{ len .Mounts }}
+LIMA_SLIRP_GATEWAY={{ .SlirpGateway }}
```

Comment:
Should be documented in internal.md

Reply:
Documented. I also changed the name to `LIMA_CIDATA_SLIRP_GATEWAY` for consistency.